src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -382,11 +382,14 @@ public static void ShowDialog(View content = null)
             var window = NUIApplication.GetDefaultWindow();
             var defaultNavigator = window.GetDefaultNavigator();
 
-            var dialog = new Dialog(content);
-            SetDialogScrim(dialog);
-
-            var dialogPage = new Page(dialog);
-            defaultNavigator.Push(dialogPage);
+            using (var dialog = new Dialog(content))
+            {
+                SetDialogScrim(dialog);
+                using (var dialogPage = new Page(dialog))
+                {
+                    defaultNavigator.Push(dialogPage);
```

Comment:
In my opinion, it is not allowed to dispose page instance after pusing it to the navigator.
I think @jaehyun0cho need to review this.

Reply:
@rabbitfor is correct.
Since the constructor of Page and Navigator.Push do not clone their parameters, using(){ } causes the given parameter objects not displayed on the screen in this case.

Consequently, using(){ } should not be used in this case.

```
// Argument dialog is not cloned in Page's constructor
// so dialogPage cannot display dialog outside of using(){ }
var dialogPage = new Page(dialog);
```
```
// Argument dialogPage is not cloned in Navigator.Push
// so Navigator cannot display dialogPage outside of using(){ }
defaultNavigator.Push(dialogPage);
```